hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/processors/CustomMessageCallProcessor.java

```diff
@@ -94,7 +93,7 @@ public CustomMessageCallProcessor(
     }
 
     /**
-     * Starts the execution of a message call based on the contract address of the given frame,
+     * Sta§rts the execution of a message call based on the contract address of the given frame,
```

Comment:
Is this change on purpose?

Reply:
Not really. My mistake ... it should have been a "$" I guess... for all the money that is going to start flowing after this fix is deployed. 💰 
I'll remove it. Thank you, Sir.